DOCS: Update en Contributor Guides (#53)
personium/new-docs: src/contributor/README.md

---
id: README

> title: Contributor Guides

Reviewer: ```suggestion
title: Contributor Guide
```

> sidebar_label: Contributor Guides

Reviewer: ```suggestion
sidebar_label: Contributor Guide
```

---

> The open source project Personium welcomes various forms of participation and contributions.

Reviewer: ```suggestion
The open source project Personium welcomes various forms of participations and contributions.
```

> ## Question

Reviewer: ```suggestion
## Questions
```

> If you have any questions about the concept and usage of Personium, please use [Slack Community](https://personium-io.slack.com/).

Reviewer: ```suggestion
If you have any questions about the concept and usage of Personium, please contact us through [Slack](https://personium-io.slack.com/).
```

> To register for the Slack community, click [here](https://bit.ly/Join_Personium_Slack).

Reviewer: ```suggestion
If you do not have a Personium Slack account, click [here](https://bit.ly/Join_Personium_Slack) to create one.
```

The usage of each channel is as follows.

| Channel name | Topic |
| ---------- | ---- |
| #general | General content |
| #random | Chat |
| #infra | Personium server construction |
| #appdev | Application development using Personium |
| #dev | Development of Personium itself |

> ## Document deficiency / improvement

Reviewer: ```suggestion
## Document maintenance and improvement
```

> Please refer to the following documents when you point out the deficiency and improve the Personium document.

Reviewer: ```suggestion
Please refer to the following guideline(s) when you want to help maintaining and improving the documents.
```

* [How to participate in document maintenance and improvement](../document-writer/README.md)

## Personium server bug report / development

> Please refer to the following documents when you report or fix bugs in Personium server software itself, or plugins or extensions.

Reviewer: ```suggestion
Please refer to the following guideline(s) when you want to report or fix bugs in Personium server software, plugins, or extensions.
```
